src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -110,9 +96,8 @@ public function getAttributeListByName(string $name): array
 
     public function getAttributes(): array
     {
-        $attributesTable = $this->getElement('attributes');
-
-        return $attributesTable->findAll('css', '[data-test-product-attribute-name]');
+        return $this->getElement('attributes')
+            ->findAll('css', '[data-test-product-attribute-name]');
```

Comment:
It is not too long I guess
```suggestion
        return $this->getElement('attributes')->findAll('css', '[data-test-product-attribute-name]');
```